modules/light-clients/06-solomachine/module.go

```diff
@@ -55,3 +66,13 @@ func (AppModuleBasic) GetTxCmd() *cobra.Command {
 func (AppModuleBasic) GetQueryCmd() *cobra.Command {
 	return nil
 }
+
+// AppModule is the application module for the Solomachine client module
+type AppModule struct {
+	AppModuleBasic
+}
+
+// NewAppModule creates a new Solomachine client module module
```

Comment:
```suggestion
// NewAppModule creates a new Solomachine client module
```
You could also call it "`06-solomachine` client module".